src/android/CameraLauncher.java

```diff
@@ -1312,20 +1312,39 @@ public void onScanCompleted(String path, Uri uri) {
     }
 
     public void onRequestPermissionResult(int requestCode, String[] permissions,
-                                          int[] grantResults) {
-        for (int r : grantResults) {
-            if (r == PackageManager.PERMISSION_DENIED) {
-                this.callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR, PERMISSION_DENIED_ERROR));
-                return;
+                                          int[] grantResults) throws JSONException {
+        String version = android.os.Build.VERSION.RELEASE;
+        if(Integer.parseInt(version) <= 13)
+        {
+            if(PermissionHelper.hasPermission(this, Manifest.permission.CAMERA)) {
```

Comment:
question: have you tested this line, because in my test in android 13 hasPermission is every time false, or you call a request to have permission, or remove this line(my android 13 worked without this if)

Reply:
With that being said, the condition seems to be doing `<=` android 13 check. But this caveat applies if your app targets `M` (API 23) or later, which is virtually all apps today.